web_src/js/features/repo-home.js

```diff
@@ -152,12 +155,14 @@ export function initRepoTopicBar() {
   });
 
   $.fn.form.settings.rules.validateTopic = function (_values, regExp) {
-    const $topics = $topicDropdown.children('a.ui.label');
-    const status = !$topics.length || $topics.last()[0].getAttribute('data-value').match(regExp);
+    const topics = topicDropdown.querySelectorAll(':scope > a.ui.label');
+    const lastTopic = topics[topics.length - 1];
+    const status = !topics.length || lastTopic.getAttribute('data-value').match(regExp);
```

Comment:
```suggestion
    const status = !topics.length || (topics[topics.length - 1]).getAttribute('data-value').match(regExp);
```

Slightly better as this should avoid accessing the array at index `-1`, but I still find this code hard to follow.